apps/docs-snippets/package.json

```diff
@@ -12,7 +12,8 @@
     "@fuel-ts/errors": "workspace:*",
     "@fuel-ts/utils": "workspace:*",
     "ethers": "^6.7.1",
-    "fuels": "workspace:*"
+    "fuels": "workspace:*",
+    "vitest": "^1.1.3"
```

Comment:
Why did you add this? Shouldn't it be globally available?

Reply:
Previously it was [failing](https://github.com/FuelLabs/fuels-ts/actions/runs/8455979446/job/23164701406#step:17:18) without it. Vitest only exposes `describe`, `expect`, `it` globally and I used `epectTypeOf`

Reply:
A bit weird that they'd expose only a part of their functionality in their "make global" opt-in feature.